nh_ews/tests/nh_clinical_frequency_ews_settings/test_set_params.py

```diff
@@ -0,0 +1,223 @@
+
+    def test_equal_minimum_and_maximum(self):
+        with self.assertRaises(ValidationError):
+            self.call_test(values=9, minimum=10, maximum=10)
+
+    def test_some_negative_values(self):
+        with self.assertRaises(ValidationError):
+            self.call_test(values=-2, minimum=-1, maximum=10)
+
+    def test_all_negative_values(self):
+        with self.assertRaises(ValidationError):
+            self.call_test(values=-11, minimum=-10, maximum=-9)
+
+
+class TestRaisesValidationErrorIfAFieldIsAboveMaximum(TestSetParams):
+    """
+    A ValidationError is raised if the config value is below the maximum
+    specified by its corresponding 'maximum' field.
+    """
+    def test_positive_values(self):
+        with self.assertRaises(ValidationError):
+            self.call_test(values=11, minimum=5, maximum=10)
+
+    def test_other_positive_values(self):
+        with self.assertRaises(ValidationError):
+            self.call_test(values=10, minimum=5, maximum=9)
+
+    def test_equal_minimum_and_maximum(self):
+        with self.assertRaises(ValidationError):
+            self.call_test(values=6, minimum=5, maximum=5)
+
+    def test_some_negative_values(self):
+        with self.assertRaises(ValidationError):
+            self.call_test(values=-1, minimum=-5, maximum=-5)
+
+    def test_all_negative_values(self):
+        with self.assertRaises(ValidationError):
+            self.call_test(values=-4, minimum=-5, maximum=-5)
+
+
+class TestDoesNotRaiseIfFieldValueIsMinimum(TestSetParams):
+    """
+    No exception is raised if the config value is equal to the minimum
+    specified by its corresponding 'minimum' field.
+    """
+    def test_positive_values(self):
+        self.call_test(values=5, minimum=5, maximum=10)
+
+    def test_other_positive_values(self):
+        self.call_test(values=9, minimum=9, maximum=10)
+
+    def test_some_negative_values(self):
+        self.call_test(values=-5, minimum=-5, maximum=1)
+
+    def test_all_negative_values(self):
+        self.call_test(values=-5, minimum=-5, maximum=-1)
+
+
+class TestDoesNotRaiseIfFieldValueIsMaximum(TestSetParams):
+    """
+    No exception is raised if the config value is equal to the maximum
+    specified by its corresponding 'maximum' field.
+    """
+    def test_positive_values(self):
+        self.call_test(values=10, minimum=5, maximum=10)
+
+    def test_other_positive_values(self):
+        self.call_test(values=6, minimum=5, maximum=6)
+
+    def test_some_negative_values(self):
+        self.call_test(values=1, minimum=-10, maximum=1)
+
+    def test_all_negative_values(self):
+        self.call_test(values=-5, minimum=-10, maximum=-5)
+
+
+class TestDoesNotRaiseIfAllParametersEqual(TestSetParams):
+    """
+    No exception is raised if all arguments are equal.
+    """
+    def test_does_not_raise_if_all_parameters_equal(self):
+        self.call_test(values=10, minimum=10, maximum=10)
```

Comment:
Break out multiple assertions in a test case

Reply:
Don't know how I missed those :confused: will fix.